app/views/simple_smart_answers/_completed_question.html.erb

```diff
@@ -1,10 +1,11 @@
-<li class="done">
-  <h3 class="question">
-    <span class="question-number"><%= completed_question_counter + 1 %></span><%= completed_question.question.title %>
-  </h3>
-
-  <div class="answer"><%= completed_question.label %></div>
-  <p class="undo">
-    <%= link_to "Change this answer", change_completed_question_path(completed_question_counter), class: "govuk-link" %>
-  </p>
-</li>
+<div class="govuk-summary-list__row">
```

Comment:
This description list feels as if it's trying to be a table - so I wonder if a table would be a more semantic way of displaying this information?

Reply:
That's what I thought too - in fact when I rebuilt `smart-answers` I did the equivalent section using a table. I was going to do the same here but then I found [some Design System guidance](https://design-system.service.gov.uk/patterns/check-answers/) on the subject.

Using a description list is also a bit more responsive - see [a current smart-answer](https://www.gov.uk/maternity-paternity-calculator/y/paternity) for the table use.

Reply:
Super - in that case I'm happy with the use of description list.